Assets/Mirror/Examples/MultipleAdditiveScenes/Scripts/PhysicsSimulator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+
+namespace Mirror.Examples.MultipleAdditiveScenes
+{
+    public class PhysicsSimulator : MonoBehaviour
+    {
+        public PhysicsScene physicsScene;
+        private float timer;
+
+        private void Awake()
+        {
+            enabled = (NetworkServer.active);
+            physicsScene = gameObject.scene.GetPhysicsScene();
+        }
+
+        void Update()
```

Comment:
Should this just use FixedUpdate instead? you would be able to get rid of the timer

Reply:
This came from [Unity's docs](https://docs.unity3d.com/ScriptReference/PhysicsScene.Simulate.html).